integration-tests/bats/sql-pull.bats

```diff
@@ -160,33 +160,47 @@ teardown() {
 }
 
 @test "sql-pull: dolt_pull force" {
-    skip "todo: support dolt pull --force (cli too)"
-    cd repo2
-    dolt sql -q "create table t2 (a int)"
-    dolt commit -am "2.0 commit"
+    cd repo1
+    dolt sql <<SQL
+SET FOREIGN_KEY_CHECKS=0;
```

Comment:
These tests probably need a little explanation as to why you are adding FK. My read is that you are disabling them so you can get the contraint violations on merge, which is great. But it takes a little puzzling out for the reader to determine that.